java/Ice/forwarder/build.gradle.kts

```diff
@@ -1,24 +1,6 @@
 // Copyright (c) ZeroC, Inc.
 
-plugins {
-    // Register the Slice tools plugin without applying it here.
-    // The plugin will be applied to all subprojects in the `subprojects` block below.
-    id("com.zeroc.ice.slice-tools") version "3.8.0-+" apply false
-}
-
 subprojects {
-    // Apply the application plugin to enable Java application support in subprojects.
-    apply(plugin = "application")
-
-    // Apply the Slice tools plugin to enable Slice compilation in all subprojects.
-    apply(plugin = "com.zeroc.ice.slice-tools")
-
-    repositories {
-        // This demo uses the latest Ice nightly build published in ZeroC's maven-nightly repository.
-        maven("https://download.zeroc.com/nexus/repository/maven-nightly/")
-        mavenCentral()
-    }
-
     tasks.withType<JavaCompile>().configureEach {
```

Comment:
Do you plan to rework the JavaCompile settings? Seems contradictory to move everything out and keep it just for this.

Reply:
Yep, these files are going away in my next PR.
Just waiting to move these last few settings until I also add support for our formatter/linter.
Since they're going to end up in the same shared build script.